Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/LATISS-Combined-Calibrations-Procedure/latiss-combined-calibrations-procedure.rst

```diff
@@ -173,7 +178,40 @@ After loading the script, a window that contains two sections, ``SCHEMA`` (top)
 - ``oods_timeout``: Timeout value, in seconds, for the Observatory Operations Data Service (``OODS``).
   Default: ``120``.
 
-An example set of configuration parameters is as follows:
+
+Configuration examples
+----------------------
+
+If no configuration parameters are passed to LOVE and the default parameters are used, the script will take 21 biases, 21 darks of 5 seconds each one, and 21 flats of 5 secnds each one. In each case, the first image will be discarded. New combined calibrations will not be generated, and verification of the images taken will be performed using the existing combined calibrations in the ``LATISS/calib`` collection (i.e., the script will do ``external verification``). In this case, no defects will be made.
```

Comment:
So this is what is mean to be run every day, correct? If so, you should say so.

Reply:
Yes. I can't tag Chris, but I'll ask him to confirm. He is one of the reviewers in Jira.  We also expect the users/observers to adjust parameters such as exposure times as needed, no?

Reply:
DMTN-222 also recommends estimating gains daily, so I'll change that in the default parameters of the script under this same ticket